.github/workflows/bundle-app.yml

```diff
@@ -0,0 +1,71 @@
+name: bundle app
+on: [push, pull_request]
+
+jobs:
+  bundle-apk:
+    runs-on: macos-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.4.0
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Ruby
```

Comment:
I believe Ruby/bundler isn't necessary for Android build.
Either changing OS to ubuntu or add special flag to avoid bundle install here to save some time.
https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/scripts/post.sh#L28

Reply:
Like skip if "${GITHUB_WORKFLOW}" == "bundle app"

Reply:
I switched from ubuntu to ios because of a "System limit for number of file watchers reached" error.

Regarding the ruby bundle thing, I squashed all commits, but I recall I had to execute bundle install.

That's also why I did https://github.com/LedgerHQ/ledger-live-mobile/pull/1437/files/2ee883b9494405fcad5876bce0dde28f5b1c7aec#diff-c7a2bbcab4d0757f7a7ba519bf73e7af3dbdb641ff4fdee2b70b684586f49cbdL25.
However it looks like I made this change when I was trying with ubuntu

Reply:
the pod install could be skip though

Reply:
However you will need it right ?

https://github.com/LedgerHQ/ledger-live-mobile/pull/1437/files/2ee883b9494405fcad5876bce0dde28f5b1c7aec#diff-c7a2bbcab4d0757f7a7ba519bf73e7af3dbdb641ff4fdee2b70b684586f49cbdL25 => I can revert that since we are using macOS but if I did it i am 99% sure it blocked when I was trying with ubuntu

Reply:
I could also switch back to Ubuntu with https://github.com/expo/expo-github-action/issues/20#issuecomment-541676895

Reply:
oh yeah, I forgot that we're using fastlane 😅  sorry!